### ISSUE TITLE
False positive for noFocusedTests

### ISSUE THREAD

**Issue by BrunoQuaresma** [biomejs/biome#6759]
### Environment information

```bash
CLI:
  Version:                      2.0.6
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           linux

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           v20.16.0
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Status:                       Not set
  Path:                         unset

Workspace:
  Open Documents:               0
```

### Rule name

noFocusedTests

### Playground link

https://biomejs.dev/playground/?code=YwBvAG4AcwB0ACAAZgBpAHQAQQBkAGQAbwBuACAAPQAgAG4AZQB3ACAARgBpAHQAQQBkAGQAbwBuACgAKQA7AAoACgBmAGkAdABBAGQAZABvAG4ALgBmAGkAdAAoACkAOwAKAGYAaQB0AEEAZABkAG8AbgAuAGYAaQB0ACgAKQA7AA%3D%3D&ruleDomains.react=recommended&ruleDomains.test=recommended

### Expected result

It should not warn the "Don't focus the test"

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by dyc3** [comment]
As a workaround, you can use overrides to only apply this rule to your test files.

**Comment by codingedgar** [comment]
> As a workaround, you can use overrides to only apply this rule to your test files.

how can i make it work only in tests?
